acceptance/tests/cloud/observability_test.go

```diff
@@ -179,14 +130,12 @@ func TestBasicCloud(t *testing.T) {
 		// TODO: Take this out
 
 		"telemetryCollector.enabled":                   "true",
-		"telemetryCollector.image":                     cfg.ConsulCollectorImage,
+		"telemetryCollector.image":                     "hashicorp/consul-telemetry-collector:0.0.1",
```

Comment:
hardcode this as test requires a collector to pass

Reply:
You shouldn't need to hardcode this as you are already passing it in as an argument to the test (like CI would)?  You should be able pull it through using the acceptance test framework flags. I think it is the only helm value that needs to be dynamic for these tests.

```
-consul-collector-image hashicorp/consul-telemetry-collector:0.0.1 
```

Reply:
oh nvm it does throw an error when you don't do it 💯 

```
Received unexpected error:
                                error while running command: exit status 1; Error: INSTALLATION FAILED: execution error at (consul/templates/telemetry-collector-deployment.yaml:2:48): telemetryCollector.image must be set to enable consul-telemetry-collector
```